paaosumfields.civix.php

```diff
@@ -6,10 +6,10 @@
  * The ExtensionUtil class provides small stubs for accessing resources of this
  * extension.
  */
-class CRM_Paaosumfields_ExtensionUtil {
+class CRM_Poaasumfields_ExtensionUtil {
```

Comment:
@lucky091588 , I am not sure I understand why you changed the spelling of the class.  It should be PAAO.

Thanks,
Steve

Reply:
Weird. I only did `civix generate:module`. I will double-check this. Thanks.

Reply:
It seems that when you do `civix generate:module` the civix will base the changes in `info.xml` `<namespace>` which has the wrong spelling. Fixing this on the latest commit.